<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>After_Splash</class>
 <widget class="QMainWindow" name="After_Splash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face Mask Detector | Menu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pict.qrc">
    <normaloff>:/newPrefix/logo.ico</normaloff>:/newPrefix/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pict.qrc">:/newPrefix/Selection_Screen.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>1164</x>
      <y>610</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Get Back To Home&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="pict.qrc">
      <normaloff>:/newPrefix/dummy.png</normaloff>:/newPrefix/dummy.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="video_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>521</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Choose Video From Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="pict.qrc">
      <normaloff>:/newPrefix/dummy.png</normaloff>:/newPrefix/dummy.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="web_cam">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Detect From Web Camera&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="pict.qrc">
      <normaloff>:/newPrefix/dummy.png</normaloff>:/newPrefix/dummy.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pict.qrc"/>
 </resources>
 <connections/>
</ui>
